chains/fragments-tests.edn

```diff
@@ -10,6 +10,16 @@
 
   {"id" 1 "jsonrpc" "2.0" "method" "system_localListenAddresses" "params" []} (ToJson) (Http.Post rpc-server)
   (FromJson) (ExpectTable) (Take "result") (ExpectStringSeq) (Log)
+  (ForEach
+   (->
+    (Log "connecting")
+    = .p2p-address
+    "swarm" >> .ipfs-args
+    "peering" >> .ipfs-args
+    "add" >> .ipfs-args
+    .p2p-address >> .ipfs-args
+    (Maybe (-> "" (Process.Run "ipfs" .ipfs-args)))))
+  ; also call connect
```

Comment:
From my experience with using the Terminal

Once you call `ipfs swam peering add <peer-address>`, `ipfs cat <cid>` and `ipfs block get <cid>` work.

So I don't undersand why you are calling `ipfs swarm connect <peer-address>` immediately below

Reply:
That was the only way to fix this situation, so there must be a reason :)